repo_logistics/README.md

```diff
@@ -0,0 +1,3 @@
+# :tent: Bit Project Bitcamp Documentation
+
+***In the `[Insert Camp Name]` directory, you will find a template subdirectory for how each Bitcamp documentation should be structured. This README.md file will go more in depth about each section.***
```

Comment:
I think we should rename the file path from [insert camp name] to sample-camp, and have it linked from the readme as well as the place where the learning lab files should be (perhaps write out the directory structure here.

Reply:
Great idea. Will be changing the file path name and adding direct links to the specific locations. 👍

Reply:
@emsesc Reminder to add instructions for how to create gifs.